trustmanager/keyfilestore.go

```diff
@@ -164,6 +211,6 @@ func listKeys(s LimitedFileStore) []string {
 }
 
 // RemoveKey removes the key from the keyfilestore
-func (s *KeyFileStore) RemoveKey(name string) error {
+func removeKey(s LimitedFileStore, name string) error {
 	return s.Remove(name)
```

Comment:
We would also need to remove the alias file here, right? The way we need to think about stuff like this makes me feel even more strongly that we should avoid adding these extra files (see comment at the top of this file).